ci/scripts/backwards-compat-test.sh

```diff
@@ -63,14 +69,69 @@ assert_eq() {
   set -e
 }
 
+################################### QUERIES
+
+run_sql () {
+    psql -h localhost -p 4566 -d dev -U root -c "$@"
+}
+
 seed_table() {
-  for i in $(seq 1 10000)
+  START="$1"
+  END="$2"
+  for i in $(seq "$START" "$END")
   do
-    run_sql "INSERT into t values ($i);" 1>/dev/null 2>&1
+    run_sql "INSERT into t values ($i, $i);" 1>$QUERY_LOG_FILE 2>&1
   done
   run_sql "flush;"
 }
 
+random_delete() {
+  START=$1
+  END=$2
+  COUNT=$3
+  for i in $(seq 1 "$COUNT")
+  do
+    run_sql "DELETE FROM t WHERE v1 = $(("$RANDOM" % END));" 1>$QUERY_LOG_FILE 2>&1
+  done
+  run_sql "flush;"
+}
+
+random_update() {
+  START=$1
+  END=$2
+  COUNT=$3
+  for _i in $(seq 1 "$COUNT")
+  do
+    run_sql "UPDATE t SET v2 = v2 + 1 WHERE v1 = $(("$RANDOM" % END));" 1>$QUERY_LOG_FILE 2>&1
+  done
+  run_sql "flush;"
+}
+
+run_sql_old_cluster() {
+  run_sql "CREATE TABLE t(v1 int primary key, v2 int);"
+
+  seed_table 1 10000
+
+  run_sql "CREATE MATERIALIZED VIEW m as SELECT * from t;" &
+  CREATE_MV_PID=$!
+
+  seed_table 10001 20000
+
+  random_update 1 20000 1000
+
+  random_delete 1 20000 1000
```

Comment:
I was thinking if the `delete`s should happen after we start the new cluster,
so we test if the new RW can successfully delete the rows that were ingested by the old cluster.

Reply:
Oh I see. I will add this as well.

Reply:
Tested here: https://github.com/risingwavelabs/risingwave/pull/10785/commits/131494e12ef8226a99b386c65adfc0edad2282ab#diff-2cd274f1db89f49e0ff90c98f5b8cbbd66241f386285b2193b5121d5c4aacbeaR261.

Just run updates and deletes after new cluster started. If no errors, test pass.